src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseStream.cs

```diff
@@ -249,6 +265,35 @@ private async Task<int> ReadAsyncCore(byte[] buffer, int offset, int count, Canc
             }
         }
 
+        private void ReadResponseTrailers()
+        {
+            // Only load response trailers if:
+            // 1. WINHTTP_QUERY_FLAG_TRAILERS is supported by the OS
+            // 2. HTTP/2 or later
```

Comment:
This (2) constraint is there simply because this work is only being done for gRPC?  Or something else?

Reply:
I thought we decided to support HTTP/1.1, for (a) consistency and (b) because it's easier to test than HTTP2.

Reply:
I double checked with WinHTTP team and `WINHTTP_QUERY_FLAG_TRAILERS ` does not support HTTP/1.1.

Reply:
+1. We should explain that in the comment.